core/runtime/src/syscalls_impl_riscv.rs

```diff
@@ -0,0 +1,82 @@
+use libtock_platform::{OneArgMemop, RawSyscalls, YieldType, ZeroArgMemop};
+
+impl RawSyscalls for crate::TockSyscalls {
+    // This yield implementation is currently limited RISC-V versions without
+    // floating-point registers, as it does not mark them clobbered.
+    #[cfg(not(any(target_feature = "d", target_feature = "f")))]
+    fn raw_yield(r0_in: YieldType) -> u32 {
+        let mut r0 = r0_in as u32;
+        let mut _a4 = 0;
```

Comment:
Why's tthis _-prefixed?

Reply:
Wait I just saw that Alistair also mentioned this. Is this a rustc bug?

Reply:
Let's merge this into Alistair's thread. I'll resolve this conversation.